pipenv/patched/notpip/_internal/build_env.py

```diff
@@ -70,22 +68,18 @@ def _create_standalone_pip() -> Iterator[str]:
 
 
 class BuildEnvironment:
-    """Creates and manages an isolated environment to install build deps
-    """
+    """Creates and manages an isolated environment to install build deps"""
 
-    def __init__(self):
-        # type: () -> None
-        temp_dir = TempDirectory(
-            kind=tempdir_kinds.BUILD_ENV, globally_managed=True
-        )
+    def __init__(self) -> None:
+        temp_dir = TempDirectory(kind=tempdir_kinds.BUILD_ENV, globally_managed=True)
 
         self._prefixes = OrderedDict(
             (name, _Prefix(os.path.join(temp_dir.path, name)))
-            for name in ('normal', 'overlay')
+            for name in ("normal", "overlay")
```

Comment:
Same as above, you replace `'` with `"` which has no real value here.
(As a side note: consistency is important, but adding this on top the 1000s of other changes makes it really hard to review.)

Reply:
Again, that is what pip did upstream:  https://github.com/pypa/pip/blob/22.0.4/src/pip/_internal/build_env.py#L78